11.md

```diff
@@ -255,6 +255,18 @@ Relays that require payments may want to expose their fee schedules.
 }
 ```
 
+### Image ###
+
+A URL pointing to an image to be used to represent the relay.
```

Comment:
It occurred to me this isn't very specific, which could cause the field to be used in competing ways by different applications. Is the use-case an icon, like I imagine? If so I would call it "icon" and recommend it should be square. That would leave room for us to include fields like "banner" in the future.

Reply:
Agreed.

Reply:
True. Will change now.